### PR TITLE
Removes an unwanted description from chicken sandwiches

### PR DESCRIPTION
**Pull request by Tsar-Salat**
## About The Pull Request
Renames the chicken sandwich desc to reference the dystopian future, instead of a political reference that makes players uncomfortable

## Why It's Good For The Game
I think we are past this

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
add: Added new things
tweak: made chicken sandwiches more dystopian and less political
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
I thought i would reference the long history of the chicken sandwiches so-called "original creator" considering how many white haired men with a thick southern accent have claimed it

**Comment by Harranhall** [comment]
What does poultry bread cage mean?

**Comment by Tsar-Salat** [comment]
> What does poultry bread cage mean?

I asked beecord #general what the description for a chicken sandwhich was.

**Comment by Harranhall** [comment]
Ah, thought it might be a pop culture reference. Will this description stand the test of time? I agree it needs to change (sheesh) but maybe something funnier? Spitballing here:
'Now with real "chicken"!'
'Factory farm to plate in no time!*'
This is a great chance to lean into the whole dystopian future thing we have going on, I agree!

________________________________

**Comment by PowerfulBacon** [comment]
Good change, but just remove the meme taste

**Comment by zeskorion** [comment]
fuck chick fil a tho

**Review by itsmeow — DISMISSED** [review]
1. do people in europe actually call this a chicken burger
2. Chik-Fil-A is so mid, let this be known, my primary goal as maintainer is to eliminate Chik-Fil-A from the codebase

File: code/modules/food_and_drinks/food/snacks_burgers.dm [hunk 1]
```diff
@@ -364,9 +364,9 @@
 
 /obj/item/reagent_containers/food/snacks/burger/chicken
 	name = "chicken sandwich" //Apparently the proud people of Americlapstan object to this thing being called a burger. Apparently McDonald's just calls it a burger in Europe as to not scare and confuse us.
-	desc = "A delicious chicken sandwich, it is said the proceeds from this treat helps criminalize homosexuality on the space frontier."
+	desc = "The so-called classic poultry bread cage. Considering how processed this is, the taste holds up pretty well."
 	icon_state = "chickenburger"
-	tastes = list("bun" = 2, "chicken" = 4, "God's covenant" = 1)
+	tastes = list("bun" = 2, "chicken" = 4, "A smiling white-haired man's deep accent" = 1)
```
Comment (PowerfulBacon) [inline code/modules/food_and_drinks/food/snacks_burgers.dm]:
```suggestion
	tastes = list("bun" = 2, "chicken" = 4)
```

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**